packages/integration-sdk-cli/src/commands/run.ts

```diff
@@ -55,7 +56,19 @@ export function run() {
       log.debug('Loading account from JUPITERONE_ACCOUNT environment variable');
       const account = getAccountFromEnvironment();
 
-      const apiBaseUrl = getApiBaseUrl({ dev: options.development });
+      let apiBaseUrl: string;
+      if (options.apiBaseUrl) {
+        if (options.development) {
```

Comment:
Super minor nit, but we could reduce the nesting by slightly refactoring to something like:

```ts
if (options.apiBaseUrl && options.development) {
  throw new Error(
    'Invalid configuration supplied.  Cannot specify both --api-base-url and --development(-d) flags.',
  );
}

const apiBaseUrl = options.apiBaseUrl || getApiBaseUrl({
  dev: options.development,
});
```

We could also create a function:

(NOTE: would be nice if `options` was typed, but we could address that later)

```ts
function getApiBaseUrlFromCliArgs(options: any): string {
  if (options.apiBaseUrl && options.development) {
    throw new Error(
      'Invalid configuration supplied.  Cannot specify both --api-base-url and --development(-d) flags.',
    );
  }

  return options.apiBaseUrl || getApiBaseUrl({
    dev: options.development,
  });
}
```